lib/index.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
+const cssNoRTLOnly = require('./rules/cssNoRTL-only.js');
 const onlySpreadCSS = require('./rules/only-spread-css');
 const noUnusedStyles = require('./rules/no-unused-styles');
 
 module.exports = {
   rules: {
+    'cssNoRTL-only': cssNoRTLOnly,
     'only-spread-css': onlySpreadCSS,
     'no-unused-styles': noUnusedStyles,
   },
 
   configs: {
     recommended: {
       rules: {
-        'react-with-styles/only-spread-css': 2,
+        'react-with-styles/cssNoRTL-only': ['error'],
```

Comment:
Hmm, what does this do without any additional config? Should this even be enabled here in the recommended config?

Reply:
Perhaps not! It does not really make sense without something passed in. Should it not be recommended then from the get-go?

Reply:
That's right, it shouldn't be in this list. This is used when setting up this eslint plugin in your project, you can enable all of the recommended configs. Since this one doesn't do anything, it should be omitted.

Reply:
Cool will remove!